app/src/main/res/values-de/androidcomponents_strings.xml

```diff
@@ -0,0 +1,260 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- The default document title of an error page (i.e., the contents of the `<title>` tag). -->
+    <string name="mozac_browser_errorpages_page_title">Seiten-Ladefehler</string>
```

Comment:
I am glad these are localised - yay!

correct me if I am wrong, @keianhzo, these XML files should stay checked in to the android-components repo, correct?